W-14792039 PDK final review (#504)
mulesoft/docs-gateway: pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-inject-parameters.adoc

async fn response_filter(state: ResponseState, RequestData(path): RequestData<String>);
----

> Instead of a function definition of defining the lambda as in the following example:

Reviewer: Maybe I'd rephrase this to a more natural expression:

"Instead of using a lambda as parameter:"

> Instead of a function definition of defining the lambda as in the following example:

Reviewer 2: I don't think the meaning of the sentence is clear. Is my guess below correct?
```suggestion
The following example demonstrates defining the lambda of a function definition:
```
